pkg/minikube/cluster/ip.go

```diff
@@ -68,12 +68,9 @@ func HostIP(host *host.Host, clusterName string) (net.IP, error) {
 		// user network case
 		if ipString == "127.0.0.1" {
 			return net.ParseIP("10.0.2.2"), nil
+		} else {
```

Comment:
lets add a comment here why this special case

Reply:
Added a comment to indicate it's for socket_vmnet network